### PR TITLE
feat : wishlist page

### PR DESCRIPTION
**Pull request by arpit01923**
-Wishlist Page

1. Wishlist page tells about products added to wishlist and having three components Navigation, Main and Footer.
2. Navigation component is used to navigate logo, search, profile, wishlist and cart page, and for login or signup.
3. Main component is used for displaying products present in wishlist.
4. Footer component is used for giving some extra information.

-What files to check :

1. src/pages/wishlist/Wishlist.jsx
2. src/pages/wishlist/main/Main.jsx

-Preview

https://user-images.githubusercontent.com/86866251/159415079-5a91b767-71a1-419f-8bcc-eca37947caef.mp4

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
-import {Home} from "./pages/home/Home"
+// import {Home} from "./pages/home/Home"
+import { Wishlist } from "./pages/pagesIndex";
 
 function App() {
   return (
     <div className="App">
-      <Home />
+      {/* <Home /> */}
+      <Wishlist />
```
Comment (ark2002) [inline src/App.js]: just a suggestion u can use the navbar and the footer component directly in the app.js if they are gonna remain the same on each of the pages .

File: src/pages/wishlist/main/Main.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+              </p>
+
+              <section class="card-footer">
+                <div class="icon">
+                  <a class="favourite" href="#">
+                    <i class="fas fa-shopping-cart"></i>Add to Cart
+                  </a>
+                </div>
+              </section>
+            </section>
```
Comment (ark2002) [inline src/pages/wishlist/main/Main.jsx]: u can make a card component out of it and call it here would help in better readability

**Review by ark2002 — APPROVED** [review]
some suggestions

File: src/App.js (hunk 1 above)
Comment (arpit01923) [reply src/App.js]: It's basically show the output, no need to check it

File: src/App.js [hunk 3]
```diff
@@ -1,11 +1,10 @@
 import "./App.css";
-import { Product } from "./pages/pagesIndex";
-
+import { Wishlist } from "./pages/pagesIndex";
```
Comment (rohanmathur91) [inline src/App.js]: No need to name index file as pagesIndex, instead just name it as index, so while importing you only have to give path till "./pages"

File: src/components/navbar/Navbar.css [hunk 4]
```diff
@@ -1,3 +1,9 @@
+<<<<<<< HEAD
+=======
+
+@import "../../App.css";
+
+>>>>>>> 6553ff6047cf854b3337c9fca60748dcf8bcae9f
```
Comment (rohanmathur91) [inline src/components/navbar/Navbar.css]: remove this

File: src/components/navbar/Navbar.css [hunk 5]
```diff
@@ -36,56 +42,62 @@
   color: var(--secondary-color);
 }
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+
+=======
+>>>>>>> b3116ee69803f3ed10585e7a7f2317945025ec3e
+=======
+>>>>>>> 6553ff6047cf854b3337c9fca60748dcf8bcae9f
```
Comment (rohanmathur91) [inline src/components/navbar/Navbar.css]: same as above

File: src/components/navbar/Navbar.css [hunk 6]
```diff
@@ -36,56 +42,62 @@
...
   right: 0.7rem;
 }
 
+<<<<<<< HEAD
+<<<<<<< HEAD
+=======
 
+>>>>>>> b3116ee69803f3ed10585e7a7f2317945025ec3e
+=======
+>>>>>>> 6553ff6047cf854b3337c9fca60748dcf8bcae9f
```
Comment (rohanmathur91) [inline src/components/navbar/Navbar.css]: same as above

File: src/pages/wishlist/Wishlist.jsx [hunk 7]
```diff
@@ -0,0 +1,13 @@
+import {Navbar, Footer} from "../../components/componentIndex"
```
Comment (rohanmathur91) [inline src/pages/wishlist/Wishlist.jsx]: Change the import as suggested above for the pages

File: src/pages/wishlist/main/dump.jsx [hunk 8]
```diff
@@ -0,0 +1,10 @@
+const wishlists = [
+  "https://images.unsplash.com/photo-1505740420928-5e560c06d30e?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8cHJvZHVjdHxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1523275335684-37898b6baf30?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NHx8cHJvZHVjdHxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1572635196237-14b3f281503f?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8NXx8cHJvZHVjdHxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Nnx8cHJvZHVjdHxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1586495777744-4413f21062fa?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8N3x8cHJvZHVjdHxlbnwwfHwwfHw%3D&auto=format&fit=crop&w=500&q=60",
+  "https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MTF8fHByb2R1Y3R8ZW58MHx8MHx8&auto=format&fit=crop&w=500&q=60",
+];
+
+export {wishlists}
```
Comment (rohanmathur91) [inline src/pages/wishlist/main/dump.jsx]: put this in another folder, say data or staticData

**Review by rohanmathur91 — APPROVED** [review]

**Pull request merged**